### PR TITLE
Doc: Set specific SMTP username

### PR DESCRIPTION
**Pull request by utsavdotpro**
## Summary of change
- https://github.com/supertokens/supertokens-node/pull/377

## Related issues
- https://github.com/supertokens/supertokens-node/issues/375

## Checklist
- [ ] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

## Remaining TODOs for this PR
NA

### CODE REVIEW

File: v2/emailpassword/email-delivery/smtp/configure-smtp.mdx [hunk 1]
```diff
@@ -118,6 +119,7 @@ init(
                             name="...",
                             email="..."
                         ),
+                        username="...", # this is optional. In case not given, from.email will be used instead
```
Comment (rishabhpoddar) [inline v2/emailpassword/email-delivery/smtp/configure-smtp.mdx:122]: this is for the python SDK, which doesn't have this setting yet. Can you please remove this?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Comment by rishabhpoddar** [comment]
Thanks @utsavdotpro we will take it from here :)

**Comment by rishabhpoddar** [comment]
I have also added your GH profile on the supertokens-core readme! :)

**Comment by utsavdotpro** [comment]
Thanks @rishabhpoddar.
I was on a short vacation so couldn't reply earlier. I am available now and happy to contribute further if you need it 😄

**Comment by rishabhpoddar** [comment]
Really appreciate your help so far :) 

You could contribute more by:
- Adding a unit test for this change. 
- Making a similar change to our python / golang repos.

**Pull request merged**